db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+from django.utils import timezone
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+    @property
```

Comment:
The `__str__` method should not be decorated with `@property`. Remove the `@property` decorator to ensure the method works correctly as a string representation method.